### ISSUE TITLE
Argentina sources are broken

### ISSUE THREAD

**Issue by rbuffat** [osmlab/editor-layer-index#1048]
The following sources are broken:
* south-america / ar / IGN orthophoto Posadas (WMTS)
* south-america / ar / IGN topographical map (TMS)
* south-america / ar / IGN topographical map (WMS)

https://www.ign.gob.ar/NuestrasActividades/InformacionGeoespacial/ServiciosOGC lists https://www.ign.gob.ar/descargas/tyc1.html as terms and conditions, but the servers report the data as CC BY 4.0: https://wms.ign.gob.ar/geoserver/ows?service=wms&version=1.3.0&request=GetCapabilities

```
<AccessConstraints>CC BY 4.0 https://creativecommons.org/licenses/by/4.0/</AccessConstraints>
```

@don-vip You seem to have looked at these datasets in https://josm.openstreetmap.de/wiki/Maps/Argentina?sfp_email=&sfph_mail=&action=diff&version=106&old_version=105&sfp_email=&sfph_mail Do you know more about the CC BY issue?

**Comment by don-vip** [comment]
> @don-vip You seem to have looked at these datasets in https://josm.openstreetmap.de/wiki/Maps/Argentina?sfp_email=&sfph_mail=&action=diff&version=106&old_version=105&sfp_email=&sfph_mail Do you know more about the CC BY issue?

Sorry I don't know anything about any source outside of France. All of the modifications I do for other countries result from lookups for similar content when an URL breaks. I don't even remember this one.

**Comment by ignaciolep** [comment]
Perhaps @Zalitoar could help. He tried to update the sources before (PR #676)

**Comment by ignaciolep** [comment]
PR #1058

**Comment by Zalitoar** [comment]
Hello world, I submitted a PR that fixes all the fallen services. 

I added privacy_policy_url and updated the license parameters also for all the services. Those new sources are highly used by the argentinean community and other folks mapping here, and provide a lot of very useful maps and photos published by the IGN (National Geographic Institute).

Thanks in advance! :)

**Comment by Zalitoar** [comment]
Hi Vincent and René, thanks for your feedback.
I made a PR which fixes the broken services and replaced others very old from the IGN. 

We also made a change in those services in order to reflect in capabilities documents the appicable legal framework of the argentine government about public data, the 27275 Law ('Right of access to public information').
This law rules the access and nature of public data produced by all the government and associates (seventh article):
https://translate.google.com/translate?hl=&sl=es&tl=en&u=http%3A%2F%2Fservicios.infoleg.gob.ar%2FinfolegInternet%2Fanexos%2F265000-269999%2F265949%2Fnorma.htm
